umm-spec-lib/src/cmr/umm_spec/umm_g/data_granule.clj

```diff
@@ -61,10 +91,9 @@
                            (map :Identifier)
                            seq)
          :production-date-time ProductionDateTime
-         :size (get (first ArchiveAndDistributionInformation) :Size)
-         :size-unit (let [SizeUnit (get (first ArchiveAndDistributionInformation) :SizeUnit)]
-                      (when (not= "NA" SizeUnit)
-                        SizeUnit))
+         :size size 
+         :size-unit (when size 
+                      "MB")
```

Comment:
does our style guide require that this be multiple lines?

Reply:
I don't know. I think either way is fine.

Reply:
I didn't see a reference to that in `https://wiki.earthdata.nasa.gov/display/CMR/Coding+Style+Guidelines`